cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySqlClient.java

```diff
@@ -0,0 +1,167 @@
+                            sourceTS.equals(targetTS));
+        } else {
+          String sourceString = rsSource.getString(currentColumnCount);
+          String targetString = rsTarget.getString(currentColumnCount);
+          Assert.assertTrue(String.format("Different values found for column : %s", columnName),
+                            String.valueOf(sourceString).equals(String.valueOf(targetString)));
+        }
+        currentColumnCount++;
+      }
+    }
+    Assert.assertFalse("Number of rows in Source table is greater than the number of rows in Target table",
+                       rsSource.next());
+    Assert.assertFalse("Number of rows in Target table is greater than the number of rows in Source table",
+                       rsTarget.next());
+    return true;
+  }
+
+  public static void createSourceTable(String sourceTable) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getCloudSqlConnection(); Statement statement = connect.createStatement()) {
+      String createSourceTableQuery = "CREATE TABLE IF NOT EXISTS " + sourceTable + "(id int, lastName varchar(255), PRIMARY KEY (id))";
+      statement.executeUpdate(createSourceTableQuery);
+
+      // Truncate table to clean the data of last failure run.
+      String truncateSourceTableQuery = "TRUNCATE TABLE " + sourceTable;
+      statement.executeUpdate(truncateSourceTableQuery);
+
+      // Insert dummy data.
+      statement.executeUpdate("INSERT INTO " + sourceTable + " (id, lastName)" + "VALUES (1, 'Priya')");
+      statement.executeUpdate("INSERT INTO " + sourceTable + " (id, lastName)" + "VALUES (2, 'Shubhangi')");
+      statement.executeUpdate("INSERT INTO " + sourceTable + " (id, lastName)" + "VALUES (3, 'Shorya')");
+
+
+    }
+  }
+
+  public static void createTargetTable(String targetTable) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getCloudSqlConnection(); Statement statement = connect.createStatement()) {
+      String createTargetTableQuery = "CREATE TABLE IF NOT EXISTS " + targetTable + "(id int, lastName varchar(255), PRIMARY KEY (id))";
+      statement.executeUpdate(createTargetTableQuery);
+//             Truncate table to clean the data of last failure run.
+      String truncateTargetTableQuery = "TRUNCATE TABLE " + targetTable;
+      statement.executeUpdate(truncateTargetTableQuery);
+    }
+  }
+
+  public static void createSourceDatatypesTable(String sourceTable) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getCloudSqlConnection(); Statement statement = connect.createStatement()) {
+      String datatypesColumns = PluginPropertyUtils.pluginProp("datatypesColumns");
+      String createSourceTableQuery = "CREATE TABLE " + sourceTable + " " + datatypesColumns;
+      statement.executeUpdate(createSourceTableQuery);
+      System.out.println(createSourceTableQuery);
```

Comment:
Remove this.

Reply:
done 
commit link- b6c555b5f19974d2b06ebeb8708160feaa988973